### PR TITLE
feat: rework index's advisory page

### PR DESCRIPTION
**Pull request by blackheaven**
I have reworked advisory page for the index:

image

---

## hsec-tools

- [x] Previous advisories are still valid

### CODE REVIEW

File: code/hsec-tools/src/Security/Advisories/Generate/HTML.hs [hunk 1]
```diff
@@ -13,32 +14,42 @@ import qualified Data.ByteString.Char8 as BS8
...
 import qualified Security.Advisories as Advisories
 import Security.Advisories.Filesystem (listAdvisories)
 import Security.Advisories.Generate.TH (readDirFilesTH)
 import Security.Advisories.Core.Advisory (ComponentIdentifier (..), ghcComponentToText)
+import qualified Security.OSV as OSV
 
 -- * Actions
 
+dbg :: IO ()
+dbg = renderAdvisoriesIndex "/home/black/ancien/haskell/security-advisories" "/home/black/ancien/haskell/security-advisories/tmp"
```
Comment (mihaimaruseac) [inline code/hsec-tools/src/Security/Advisories/Generate/HTML.hs:51]: There are some hardcoded paths here

**Review by mihaimaruseac — APPROVED** [review]
I like it!

File: code/hsec-tools/src/Security/Advisories/Generate/HTML.hs (hunk 1 above)
Comment (blackheaven) [reply code/hsec-tools/src/Security/Advisories/Generate/HTML.hs:51]: Sh*t my debug code, sorry for that

**Comment by blackheaven** [comment]
Fixed:

* Dead/debug code, thanks @mihaimaruseac 
* Spacing, thanks @tchoutri

**Review by TristanCacqueray — APPROVED** [review]
Well done!

**Review by mihaimaruseac — APPROVED** [review]

**Comment by frasertweedale** [comment]
Very nice!  Thanks you @blackheaven and reviewers!

**Pull request merged**